CMakeLists.txt

```diff
@@ -117,6 +113,8 @@ else()
 endif(USE_MPI)
 
 if(USE_CUDA)
+    SET(CMAKE_CUDA_HOST_COMPILER "${CMAKE_CXX_COMPILER}")
```

Comment:
Suggested [here](https://www.jetbrains.com/help/clion/cuda-projects.html#custom-host-compiler) way to set custom compiler via two lines
```
set(CMAKE_CUDA_HOST_COMPILER "${CMAKE_CXX_COMPILER}")
set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -ccbin ${CMAKE_CXX_COMPILER}")
```
results in 
```
nvcc fatal : redefinition of argument 'compiler-bindir'
```
with old CUDA version. So I believe setting only `CMAKE_CUDA_HOST_COMPILER` is enough:
> This maps to the `nvcc -ccbin` option.
    https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_HOST_COMPILER.html